### ISSUE TITLE
Earmark option

### ISSUE THREAD

**Issue by LostKobrakai** [boydm/phoenix_markdown#6]
If I put the earmark config of the readme in my config.exs I get the following error:

```
** (Mix.Config.LoadError) could not load config config/config.exs
    ** (CompileError) config/config.exs:18: Earmark.Options.__struct__/1 is undefined, cannot expand struct Earmark.Options
```

Maybe phoenix markdown could use `struct!/2` so the config does only need to be a map.

**Comment by boydm** [comment]
I'll look into that.

**Comment by boydm** [comment]
Yes. Good catch. The tests were passing in the full struct, but you are right that it doesn't work correctly from the config file.

This is now fixed in master. Will probably push a .1 later today to hex.

**Comment by boydm** [comment]
Fixed and pushed to hex as 1.0.1
